pkg/api/types.go

```diff
@@ -1763,7 +1767,9 @@ type ServiceSpec struct {
 	// This field will be ignored if the cloud-provider does not support the feature.
 	LoadBalancerIP string `json:"loadBalancerIP,omitempty"`
 
-	// Optional: Supports "ClientIP" and "None".  Used to maintain session affinity.
+	// Optional, used to maintain session affinity.
+	// Supports "ClientIP", "RequireNodeLocal" and "None". The second makes
+	// most sense with `Type: ClusterIP`. Defaults to None.
```

Comment:
Describe the ClientIP and RequireNodeLocal affinity here so it shows up in swagger, especially what happens if there is no local service (indicate whether the service will be host unreachable, fallback to a remote, etc).  I assume in the future we will have `PreferNodeLocal` just from the description, but if that's not the case then the name may be better as `NodeLocal`.